test/lib/generate/cjs-test.ts

```diff
@@ -68,7 +68,7 @@ describe('generate (cjs)', function () {
         'cjs-main-file-does-not-exist'
       );
       await expect(generate(FIXTURE_PATH)).rejects.toThrow(
-        'Unable to determine plugin entry point.'
```

Comment:
I tried to construct a `package.json` test case to throw the existing default error... and couldn't figure it out.

Reply:
I'll try to take a look at this so we can test the old exception still.